examples/skaffold/README.md

```diff
@@ -0,0 +1,138 @@
+# Multiple Cloud Functions, Same Host
+
+## Introduction
+
+This example shows you how to deploy multiple Cloud Functions to a single host.
```

Comment:
This example shows how to develop multiple functions on a Kubernetes cluster with a single host.

- We're not using the Cloud Functions product anymore (FF's can be deployed to many environments).
- This seems like an opinionated way for a multiple function, single host app. Let's label that, as we may want a simpler multiple function, single host app in a different example.